### DISCUSSION TITLE
Set dark mode as default

### DISCUSSION THREAD

**Discussion by whitebear13579** [LekoArts/gatsby-themes#1011] [Questions]
When users browse the web, the light theme is loaded by default
How can I set the dark mode as the default and turn off the button to switch between light and dark modes?

**Comment by LekoArts** [comment]
> How can I set the dark mode as the default

Change the `initialColorModeName` name in the Theme UI configuration

> turn off the button to switch between light and dark modes

Shadow the specific component that contains the toggle and remove it

**Reply by yiyaz** [reply]
The problem with just setting dark mode as default is that the site automatically detects if my device is on light/dark mode and adjusts to it. So it will not have an impact on those devices. I think what OP wanted was to have dark mode on always (at least I would be curious to know how to implement that). 
Thanks!

**Reply by LekoArts** [reply]
Then set `useColorSchemeMediaQuery` to `false`: https://theme-ui.com/color-modes#responding-to-the-prefers-color-scheme-media-query
